keyboards/dz60/keymaps/split_space_arrows/rules.mk

```
@@ -0,0 +1,7 @@
+# Build Options
+# override the options specified in dz60/rules.mk
+
+BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration(+1000)
```

We're trying to get rid of these size impact numbers; they're seldom if ever accurate.

```suggestion
BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration
```